great_expectations/render/renderer/site_builder.py

```diff
@@ -112,6 +117,10 @@ def build(cls, data_context, site_config, specified_data_asset_name=None):
 
             for run_id, v0 in data_context.list_validation_results(validations_store=site_config['profiling_store']).items():
                 for datasource, v1 in v0.items():
+
+                    if datasource not in datasources_to_document:
+                        break
```

Comment:
should this be continue?

Reply:
Of course! Thank you @jcampbell ! - pushed.